### PR TITLE
Narrow peak debug

### PR DESCRIPTION
**Pull request by LeilyR**
fixed a bug in narrowPeaks which brought to us by #120

### CODE REVIEW

File: pygenometracks/tracks/BedGraphTrack.py [hunk 1]
```diff
@@ -174,7 +175,7 @@ def get_scores(self, chrom_region, start_region, end_region, return_nans=True):
                                      "chromosome name inside the bedgraph "
                                      "file. This will generate an empty "
                                      "track!!\n")
-                    return
+                    return score_list, pos_list
```
[the remark this answers is not in the document]
Comment (LeilyR) [reply pygenometracks/tracks/BedGraphTrack.py]: right, I need to change that error message to warning too.

**Comment by LeilyR** [comment]
> Hi,
> I think we should agree on the return of `get_score`.
> Initially, when the chromosome was not on the file it was raising an error and when it was part of the file but there was no item in the region it was returning [nan], (start_region, end_region).
> I guess this was to be able to make bedgraph to plot correctly.
> So, if we keep this behaviour, when the chromosome is not part of the file, it should return [nan], (start_region, end_region).
> 
> The class which use this function are:
> bedgraph ok
> bedgraphmatrix ok
> epilogos ok
> narrowPeaks.
> In the case of narrowPeaks, `for idx, peak in enumerate(score_list):` will go once except if we test before that the score_list is not [nan]...

I will check for them, thanks!

**Comment by lldelisle** [comment]
Hi,
Sorry, it is a misunderstanding. As I saw that you used `return` and in plot you used:
```
        if pos_list == []:
            return
```
I thought you would prefer that the default is an empty list.
No problem to put it back to nan. Do you want to do it or I make PR?

**Comment by LeilyR** [comment]
Either is fine by me, I would leave it be as long as it works fine. Thanks for the clarification!

**Comment by lldelisle** [comment]
I prefer to let it empty this way we do not need to import numpy to check if it is equal to `np.nan`...

**Pull request merged**